### ISSUE TITLE
GDrive install error

### ISSUE THREAD

**Issue by FogShard** [Qengineering/RPiMotionCam#5]
Hello, 

Just like wytr before me, I am stuck on step 20 of the wiki guide for adding Google Drive API. 
I have followed the instructions as listed, and receive the same error:
```
pi@rpizero2w:~ $ gdrive account add
No valid arguments given, use 'gdrive help' to see available commands

``` 
Following your advice to wytr, I attempted to build gdrive from scratch (gdrive was not listed in/under [$ ls] to begin with.) 
After selecting gdrive, and entering the command 
```
$ cargo build --release

``` 
as prompted, the following error occurs: 

```
error: failed to get 'google-drive3' as a dependency of the package 'gdrive v3.9.1' (/home/pi/gdrive)'

caused by:
  failed to load source for dependency 'google-drive3'

caused by:
  unable to update https://github.com/prasmussen/google-apis-rs?branch=resumable-fix#lf754445

caused by:
  failed to fetch into: /home/pi/ .cargo/git/db/google-apis-rs-9de08c63e04c0b9f

caused by:
  network failure seems to have happened
  if a proxy or similar is necessary 'net.git-fetch-with-cli' may help here
  https://doc.rust-lang.org/cargo/reference/config.html#netgit-fetch-with-cli

caused by:
  SSL error: received early EOF; class=Ssl (16); code=Eof (-20)
```

**Comment by Qengineering** [comment]
Do you use the Raspberry Pi Zero2?
I've update the RPi4 version. However didn't find the time for the Zero2.
I will immediately make up for the loss.
Once ready, you're the first to know.

**Comment by FogShard** [comment]
Yes, I am setting up a home surveillance network using 4 raspberry pi zero 2 W. My plan is to install this on each of them.

Thank you for your time, I look forward to your update

**Comment by Qengineering** [comment]
Please download [the NEW version](https://ln5.sync.com/dl/cc6b325c0#h39wmaf2-5tsnvseq-n9ruj7nz-kp2i7eyr).

**Comment by FogShard** [comment]
I can confirm the new Zero2W update is a success. I successfully flashed, installed and ran the new version on both of my Zero2W's.

Now I have another issue: I have set up overlay as per your settings wiki, and I successfully receive email notifications for recordings, but no video files are being sent to my dedicated Drive account.

Is there anything I could have overlooked?

**Comment by Qengineering** [comment]
By enabling the overlay, videos are temporarily stores in RAM before uploading to Gdrive.
Most likely the limited size of RAM on the RpiZ2 (500 MB) is casing your issue. 
I will look into the matter. Although, I'm afraid this can't be solved.
